iree/compiler/Dialect/Flow/Transforms/test/dispatch_linalg_on_tensors.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: iree-opt -split-input-file -verify-diagnostics -iree-flow-dispatch-linalg-on-tensors-pass -iree-flow-dispatch-linalg-on-tensors-tile-sizes="1,2" -iree-flow-dispatch-linalg-on-tensors-enable-fusion-always -canonicalize -cse %s | IreeFileCheck %s
+// RUN: iree-opt -split-input-file -verify-diagnostics -iree-flow-dispatch-linalg-on-tensors-pass -iree-flow-dispatch-linalg-on-tensors-tile-sizes="1,2" -canonicalize -cse %s | IreeFileCheck %s
```

Comment:
Actually this file is mostly legacy. Maybe just worth deleting it? EVerything here is superceded by `dispatch_linalg_on_tensors_dynamic.mlir`.

Reply:
+1. A few tests are indeed duplicate. I separated the clean up in another pull request: https://github.com/google/iree/pull/5246. Reverted the change here.